js/app/modules/layout/articleStack.js

```diff
@@ -239,8 +239,10 @@ const ArticleStack = new Module.Class({
 
     _on_history_changed: function () {
         let item = HistoryStore.get_default().get_current_item();
-        if (item.page_type !== Pages.ARTICLE)
+        if (item.page_type !== Pages.ARTICLE) {
+            this.get_children().forEach((view) => view.set_active(false));
```

Comment:
We also need to call view.set_active(true) the the current view dont we?
even if there is no current user for it